stable/sonarqube/values.yaml

```diff
@@ -0,0 +1,57 @@
+# Default values for sonarqube.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: sonarqube
+  tag: 6.5
+service:
+  name: sonarqube
+  type: LoadBalancer
```

Comment:
I don't think `LoadBalancer` should be the default.

Reply:
I prefer `LoadBalancer` myself. There was a similar conversation when merging `sonatype-nexus` (see comments on the `values.yaml` file): https://github.com/kubernetes/charts/pull/1915/files/6b77974124c2781ac5d2ee4b7c2c61f76ee62920#diff-2ad2263f9384cb405ca5a6e750e2b51a

Unless there is a standard for this that I don't know about, could we please go forward with it this way?

Reply:
OK, let's keep it then. BTW, I notice there's an incorrect `app` label on the PVC in the Nexus chart. Care to fix it? https://github.com/kubernetes/charts/blob/master/stable/sonatype-nexus/templates/pvc.yaml#L7